CRVReco/fcl/prolog_v11.fcl

```diff
@@ -372,6 +372,8 @@ BEGIN_PROLOG
       useNoFitReco                     : true
       usePEsPulseHeight                : false  //using the PEs that were calculated using the pulse height instead of pulse area
       bigClusterThreshold              : 800
+      fiberSignalSpeed                 : 140    //140 mm/ns  //FIXME: The correct value should be 175 mm/ns.
```

Comment:
As discussed with Ralf, this number should be compared with the liturature.

Reply:
> As discussed with Ralf, this number should be compared with the liturature.

Is it important to do this before we start the upcoming production?

Reply:
The signal speed of 140 mm/ns is consistent with the way the signals are simulated in the current setup. Using a different value would cause problems with the reconstruction of the longitudinal position of the hits along the counter. 
However, we measured a value of 175 mm/ns at a previous testbeam, and the CRV counter simulation (with its lookup tables) originally also showed a signal speed of 175 mm/ns. Something must have changed in the simulation, but I haven't been able to find out the reason. Also - as Yuri pointed out - this value of 175 mm/ns seems high compared to other literature values.
I think we should keep the current value of 140 mm/ns for the upcoming production, but still need to solve this problem.